woocommerce/navigation
# Allow multiple menus and flatten menu items

Flattens the menu items for easier parsing and allows menu categories and items to easily specify an ID (`primary` by default).

Also moves some of the existing items to the `secondary` menu.  Follow-up PRs will be needed to separate the profiler and fix several unrelated erroneous items under the secondary menu.

### Testing

1.  Check that all items are under the `window.wcNavigation.menuItems` global and include a `menuId` property for each item.
1. Check that using the selector `getMenuItems( 'menu-id' )` retrieves the correct items.  You can see this in React dev tools under the `Navigation` component
1. Make sure all tests are passing.

<img width="685" alt="Screen Shot 2020-07-23 at 2 48 32 PM" src="https://user-images.githubusercontent.com/10561050/88283235-b4c2b580-ccf3-11ea-8afd-26e977eb2fdf.png">
